drupal.make.yml

```diff
@@ -0,0 +1,46 @@
+core: 7.x
+api: 2
+
+# Common attributes.
+defaults:
+  projects:
+    subdir: "contrib"
+
+projects:
```

Comment:
I'd trim the fat here to what you really need.  However we can do that in a future PR.